### PR TITLE
Add linters in golangci.yml (we can make up linters one by one)

### PR DESCRIPTION
**Pull request by wuyingjun-lucky**
Signed-off-by: wuyingjun <wuyingjun_yewu@cmss.chinamobile.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -2,7 +2,28 @@ run:
...
+    - goimports
+    - gosimple
+    - govet
+    - misspell
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unused
+    - varcheck
+    - whitespace
```
Comment (Iceber) [inline .golangci.yml:21]: It seems to ignore the [ineffassign](https://golangci-lint.run/usage/linters/#enabled-by-default) that are enabled by default

Comment (wuyingjun-lucky) [reply .golangci.yml:21]: https://golangci-lint.run/usage/linters/ 
yes , I missed it

Comment (wuyingjun-lucky) [reply .golangci.yml:22]: done

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**